### ISSUE TITLE
Ability to detach a tab

### ISSUE THREAD

**Issue by Zakarisz** [christopho/solarus-quest-editor#283]
It would be useful to be able to detach a tab (move to a new window).
One window with a store map on a screen, one window with the real map to edit on another screen.

--

Ce serait assez utile de pouvoir détacher un onglet (l'ouvrir dans une nouvelle fenêtre) quand on travaille avec 2 écrans. Un écran avec une map utilitaire, et l'autre avec la map en cours de création.

**Comment by Diarandor** [comment]
Note that you can already open the Quest Editor twice, or as many times as you want. You can use this to work on different things on each screen.

WARNING: be careful, because if you work on the same thing on both screens (by mistake), you could overwrite your current work and lose it. This can be avoided just closing some tabs, so that you have different things opened for each window.

**Comment by oclero** [comment]
@Diarandor your solution is too dangerous, you can override project.db if you save in the wrong window !!! I strongly discourage people to do that ^^

**Comment by Diarandor** [comment]
You are right, that is very risky. Actually, I only do that when I need to open two different projects simultaneously, and in that case it is safe.
